src/components/List.js

```diff
@@ -1,19 +1,18 @@
 import React, { useState, useEffect } from 'react'
 
 const List = ({ multiplyNumbers }) => {
-    const [items, setItems] = useState([]);
+  const [items, setItems] = useState([]);
+  useEffect(() => {
+    // Implementation goes here
```

Comment:
Might be a good idea to leave a comment here to make it both easier to search and to let the user know we're waiting for our implementation.